src/main/java/edu/gatech/cs2340/trydent/animation/Keyframe.java

```diff
@@ -0,0 +1,43 @@
+package edu.gatech.cs2340.trydent.animation;
+
+import edu.gatech.cs2340.trydent.math.BaseVector;
+import edu.gatech.cs2340.trydent.math.Orientation;
+import edu.gatech.cs2340.trydent.math.Position;
+import edu.gatech.cs2340.trydent.math.Scale;
+import edu.gatech.cs2340.trydent.math.curve.Interpolation;
+
+/**
+ * Information holder class for a single keyframe. Here as an inner class
+ * for encapsulation.
+ */
+public class Keyframe extends Orientation {
```

Comment:
I think the name collision isn't a problem, since users should never work with Keyframes directly. They are only constructed by the KeyframeAnimationBuilder.